### DISCUSSION TITLE
What does Node  ConditioningCombine Do?

### DISCUSSION THREAD

**Discussion by Gasskin** [Comfy-Org/ComfyUI#229] [Q&A]
following the comfyui example,if i want to mix 2 controlnet (for example,if i want mix 2 canny controlnet

i will do like:

![WNF KMN_9 03_9 JOW2 WH](https://user-images.githubusercontent.com/45622493/227101901-42dfb1df-feb1-4fdb-9ccf-da3b7b8d765d.png)

and the output result is close to what i imagined

but if i use the node ConditioningCombine like:

![{W4ZJWQ40_23F_F5GRXC1I](https://user-images.githubusercontent.com/45622493/227102412-beed1a25-4064-452b-a4f6-eb79eafb69b6.png)

the result is strange,just like only use the condition_1

in short,ConditioningCombine's result is totally different from the way that connect 2 ControlNetApply together

maybe i am using it incorrectly,are these two ways the samething?

**Comment by comfyanonymous** [comment]
At every step it will "execute" both prompts and then average out the outputs. It's like AND on the a1111 ui if you are familiar with that.
